tango_simlib/tango_sim_generator.py

```diff
@@ -223,6 +215,7 @@ def attr(tango_device_instance, new_val):
         fget=TangoTestDeviceServerStaticAttrs.read_fn,
         fset=TangoTestDeviceServerStaticAttrs.write_fn,
     )
+    attr.setter(TangoTestDeviceServerStaticAttrs.write_fn)
```

Comment:
@tockards Have you discovered why this is necessary after futurize stage 2?

Reply:
Nope, I have not yet discovered why it is necessary. I am currently reading the differences between `python 2` and `python 3` w.r.t `metaclasses` and `object`